### ISSUE TITLE
Update Proxy Charts to be compliant with PostgreSQL 

### ISSUE THREAD

**Issue by mlycore** [apache/shardingsphere-on-cloud#243]
### Request description

Currently the Proxy charts are tested with MySQL. However ShardingSphere could also run with PostgreSQL. So we need to update the charts and make sure it can start a ShardingSphere cluster with PostgreSQL config.

### Environment

- your ShardingSphere-Operator Version
- Kubernetes distro (e.g. RKE/K3s/EKS/OpenShift) and version

**Comment by yikuaibro** [comment]
I'm working on it. Please assign me.

**Comment by mlycore** [comment]
Thanks ! @yikuaibro

**Comment by yikuaibro** [comment]
Hi@mlycore 
 I have completed the test of this issue, the general work is as follows:
1. Confirmed that both proxy helm and operator support pgsql startup and run according to the pg protocol. After modifying the values and deployment in the Proxy Charts, the ```helm proxy charts``` successfully pulled up pgsql, and after mapping the proxy to the local, psql successfully passed the proxy pairing logic. The library performs normal library, table operations, inserting data, etc.
2. Both proxy helm and operator can be downloaded according to the pg driver version, which is the same as the mysql driver, currently postgresql-42.4.1.jar
